### ISSUE TITLE
Question: explanations for the loss going down and up

### ISSUE THREAD

**Issue by CharlieCheckpt** [bytedance/ibot#31]
Hello,

I was looking at the training curves for the log of you provided, and we can see that the training loss : 
1 ) goes down very quickly from epoch 0 to 50  
2 ) goes up from epoch 50 to 200
3 ) goes down again slowly from epoch 200 to 400

<img width="762" alt="image" src="https://user-images.githubusercontent.com/10373261/216765300-d081bbab-c9f7-4444-a6ee-ad912afd961b.png">

1 ) Could be explained by the warmup of the learning rate I guess ? But I don't understand 2) and 3). 
Specifically, the loss is actually lower at 50 epochs than at 400 epochs. So does the model benefit from being trained more than 50 epochs ?

**Comment by jensenzhoujh** [comment]
Hi @CharlieCheckpt,

Such a trend is fairly common for non-contrastive objectives. At early epochs, it's easy for models to find trivial solutions that favor the CE measurement. The stage where the training loss rises is when the sharpness and smoothness constraints dominate the training. This doesn't mean the representation becoming worse, as indicated by the monotonic rise of training NMI, ARI, and F-score.
